site/docs/mark/errorband.md

```diff
@@ -22,12 +23,14 @@ Error band in Vega-Lite can either be used to aggregate raw data or directly vis
 To create an error band, set `mark` to `"errorband"`.
 
 ## Documentation Overview
+
 {:.no_toc}
 
 - TOC
-{:toc}
+  {:toc}
```

Comment:
why?

Reply:
This is what prettier does for me. Should I revert back?

Reply:
Please rebase so you have the right format.